overlord/snapstate/snapstate.go

```diff
@@ -1909,6 +1917,16 @@ func Remove(st *state.State, name string, revision snap.Revision, flags *RemoveF
 		if tp, _ := snapst.Type(); tp == snap.TypeApp && removeAll {
 			ts, err := AutomaticSnapshot(st, name)
 			if err == nil {
+				if sz, err := EstimateSnapshotSize(st, name); err == nil {
+					requiredSpace := requiredSpaceWithMargin(uint64(sz))
```

Comment:
it's a bit unclear why use the 10% margin in this case?

Reply:
Right, I was trying to be super careful, but given that estimation is already pessimistic/simplistic, margin doesn't make sense here. Will remove.